### PR TITLE
DROTH-246 (16) Vehicle limitations to have free text field for additional information

### PR DESCRIPTION
**Pull request by marquesfm**
- Modifications in Prohibitions Form for have the new field;
- Add additionalInfo at the class ProhibitionValue;
- Add Collums at Prohibition Value Table;
- Modifications in querys for have new field, fetchProhibitions and insertProhibitionValue methods;
- Unit tests for all modifications.

### CODE REVIEW

**Comment by leirasf** [comment]
Looks good to me

File: UI/src/view/ProhibitionFormElements.js [hunk 1]
```diff
@@ -163,12 +177,24 @@
             '</div>';
         }
 
+
+        function additionalInfomationField(prohibition) {
+
+          return '' +
+              '<label>Tarkenne:</label>' +
```
Comment (halied) [inline UI/src/view/ProhibitionFormElements.js]: Is it possible to have Tarkenne label and text field to be indented to the same level as Rajoitus ei koske... and dropdown menu before it?

**Comment by halied** [comment]
One comment added, otherwise looks ok.

File: UI/src/view/ProhibitionFormElements.js (hunk 1 above)
Comment (marquesfm) [inline UI/src/view/ProhibitionFormElements.js]: Modifications suggested Done Hanna.

**Pull request merged**